hack/build/bazel-build-images.sh

```diff
@@ -31,5 +31,5 @@ for tag in ${docker_tag}; do
         --define container_prefix=${docker_prefix} \
         --define container_tag=${tag} \
         --host_force_python=PY3 \
-        //:test-container-images //cmd/cdi-operator:cdi-operator-image //cmd/cdi-controller:cdi-controller-image //cmd/cdi-apiserver:cdi-apiserver-image //cmd/cdi-cloner:cdi-cloner-image //cmd/cdi-importer:cdi-importer-image //cmd/cdi-uploadproxy:cdi-uploadproxy-image //cmd/cdi-uploadserver:cdi-uploadserver-image //cmd/ovirt-populator:ovirt-populator-image
```

Comment:
how come this fails? are we really okay with removing?

Reply:
"make all" is broken without it.  Remnant of forklift pr.  There is a bogus target listed there